jans-cedarling/bindings/cedarling_python/PYTHON_TYPES.md

```
@@ -1,45 +1,78 @@
+
 # Cedarling Python bindings types documentation
 
 This document describes the Cedarling Python bindings types.
 Documentation was generated from python types.
 
-# AuthorizeResult
+AuthorizeMultiIssuerRequest
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)